lyrebird/mock/blueprints/apis/config.py

```diff
@@ -19,3 +22,21 @@ def put(self):
             return context.make_ok_response()
         except Exception as e:
             return context.make_fail_response(str(e))
+
+    def patch(self):
+        try:
+            update_conf = request.get_json()
+            if update_conf is None:
```

Comment:
from lyrebird import log

logger = log.get_logger()

if not update_conf:
    logger.warning('Requests body is empty')
    return make_ok_response()
if not isinstance(update_conf, dict):
    return make_fail_response('Request body must be JSONObject!')